thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/route.js

```diff
@@ -55,22 +107,25 @@ export default Ember.Route.extend({
     },
 
     /**
-    * Trigger alert replay sequence after create has finished
-    * @param {Number} alertId - Id of newly created alert function
+    * Trigger onboarding sequence starting with alert creation. Once triggered,
+    * we must look up the new alert Id as confirmation.
+    * @param {Object} data - contains request query params for alert creation job
     * @method triggerReplaySequence
     */
-    triggerReplaySequence(alertId) {
-      const replayUrl = this.controller.buildReplayUrl(alertId);
-      fetch(replayUrl, postProps('')).then(checkStatus)
-        .then((replayId) => {
-          this.transitionTo('manage.alert', alertId, { queryParams: { replayId }});
+    triggerOnboardingJob(data) {
+      const onboardUrl = `/detection-onboard/create-job?jobName=${data.jobName}&payload=${encodeURIComponent(data.payload)}`;
+      const newName = JSON.parse(data.payload).functionName;
+      let onboardStartTime = moment();
```

Comment:
is this variable used anywhere?

Reply:
Ah - yes. I'm setting a time marker when onboarding begins here, and measuring elapsed time later, when checking on create and replay status.

Reply:
Yes, we use it to measure time passed from onboarding start during alert create confirmation loop.